doc/ems-application-guide/src/ems-actuators/thermal-envelope.tex

```diff
@@ -55,6 +55,20 @@ \subsection{Surface Construction State}\label{surface-construction-state}
 
 \subsection{Surface Boundary Conditions}\label{surface-boundary-conditions}
 
+Three actuators, called ``Surface,''~ are available for controlling the temperature and wind speed used for ``Outdoors''~ boundary conditions during heat transfer calculations at the outside face of surfaces. Each heat transfer surface will make available actuators with the following three control types:
+
+\begin{itemize}
+\item
+  ``Outdoor Air Drybulb Temperature.''~ This is the dryblub temperature of the ambient outdoor air exposed to the surface, in degrees C. This is the temperature used for surface convection heat transfer boundary conditions on the outdoor, outside-face, of the surface exposed to exterior environment (unless it is raining).
+\item
+  ``Outdoor Air Wetbulb Temperature.''~ This is the wetbulb temperature of the ambient outdoor air exposed to the surface, in degrees C. When the weather indicates that it is raining outside, the surface convection heat transfer boundary conditions use the wetbulb temperature rather than the drybulb temperature (along with a high convection heat transfer coefficient). This override allows changing the outdoor wetbulb experienced by the surface exposed to exterior environment (when it is raining).
+\item
+  ``Outdoor Air Wind Speed.''~ This is the wind speed of the outdoor air exposed to the surface, in m/s. This is the velocity used for some surface convection heat transfer correlations for the outside-face of surfaces exposed to wind. 
+\item
+  ``Outdoor Air Wind Direction.''~ This is the wind direction of the outdoor air exposed to the surface, in degree. This is the direction used for some surface convection heat transfer correlations for the outside-face of surfaces exposed to wind.
+\item
+  ``View Factor To Ground.''~ This is the view factor to ground of the outdoor air exposed to the surface. 
```

Comment:
Would be good to mention here that this view factor is used for both solar and radiant exchange (if I remember correctly) and to mention the best calling point to use. `BeginZoneTimestepAfterInitHeatBalance` is that last calling point before `InitSurfaceHeatBalance` is called where the actuators are used.  `BeginZoneTimestepBeforeInitHeatBalance` could also be used.

Reply:
I realized that I used `BeginTimestepBeforePredictor` in my test file. Does it have to be either `BeginZoneTimestepAfterInitHeatBalance` or `BeginZoneTimestepBeforeInitHeatBalance`?

Reply:
If I'm following everything correctly, with `BeginTimestepBeforePredictor` I think you'll find that the actual impact on the view factor will lag the EMS action by one zone timestep.

Reply:
Aha, I see. I probably didn't see the impact because I only considered a seasonal shading coefficient variation. If I considered hourly variation (during the daytime), I would probably see some differences.